qasc/evaluator/evaluator.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+import csv
+from typing import *
+import logging
+import sys
+import json
+
```

Comment:
I am assuming this code is exactly same, so won't review it again.

Reply:
Indeed, nothing changed in this code.